modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -66,6 +66,21 @@ def expired?
       where.not(resolution: { resolving_type: 'AccreditedRepresentativePortal::PowerOfAttorneyRequestExpiration' })
     }
 
+    scope :for_user, lambda { |user|
+      for_power_of_attorney_holders(
+        user.power_of_attorney_holders
+      )
+    }
+
+    scope :for_power_of_attorney_holders, lambda { |poa_holders|
+      names = PowerOfAttorneyHolder::PRIMARY_KEY_ATTRIBUTE_NAMES
+      values = poa_holders.map do |poa_holder|
+        poa_holder.to_h.values_at(names)
+      end
+
+      where(names => values)
+    }
```

Comment:
@stevenjcumming This two new model scopes not exercised by the tests in this PR, and indeed they are not correct yet (e.g. should be `values_at(*names)`.

These are corrected and consumed in the PR that comes right after this though!